cli/packages/prisma-client-lib/src/Client.ts

```diff
@@ -423,6 +424,14 @@ export class Client {
 
     const type = this.getDeepType(field.type)
 
+    if (isRelayConnection) {
+      let relayConnectionHasScalars = false
+      relayConnectionHasScalars = connectionNodeHasScalars({ type })
```

Comment:
Please turn this into `const relayConnectionHasScalars = connectionNodeHasScalars({ type })`.